### PR TITLE
Chapter2/homework/1

### PR DESCRIPTION
**Pull request by jeanroyce**
I removed `Barbie Pankhurst.` from all test files.

### CODE REVIEW

**Comment by jeanroyce** [comment]
> Instead of working with strings (the way you did in chapter1), try working with objects. Convert the parsed lines into people and then process people and not strings.

Done!

File: Src/BootCamp.Chapter/BalanceStats.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class BalanceStats
+    {
+        public static string FindHighestBalanceEver(string[] peopleAndBalances)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/BalanceStats.cs:10]: You're still working with string[]. That needs to be changed to Person[].

**Comment by jeanroyce** [comment]
BalanceStats now only process people (object), I think 😁

File: Src/BootCamp.Chapter/Person.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+namespace BootCamp.Chapter
+{
+    public class Person
+    {
+        private readonly string _name;
+        public string GetName()
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Person.cs:6]: Use properties and not getter methods!

**Comment by Almantask** [comment]
Looks good now. But if you can put logic into models in isolation, you should.

**Pull request closed without merging**